src/Summoner/Project.hs

```diff
@@ -71,7 +73,7 @@ generateProject projectName Config{..} = do
     let settingsWarnings = cWarnings
 
     putTextLn $ "The project will be created with the latest resolver for default GHC-" <> showGhcVer defaultGHC
-    settingsTestedVersions <- sortNub . (defaultGHC :) <$> case cGhcVer of
+    settingsTestedVersions <- NE.sortWith id . nub . (defaultGHC :|) <$> case cGhcVer of
```

Comment:
You can use `sort` function for `NonEmpty`. And, also, let's make call to `nub` qualified, so it will be something like:

```haskell
NE.sort . NE.nub
```